### ISSUE TITLE
integrate frontend with real backend api

### ISSUE THREAD

**Issue by davemfish** [natcap/urban-online-workflow#27]
This is on hold until after #22 and maybe after their are backend implementations for running wallpapering and running invest.

**Comment by davemfish** [comment]
We could consider updating to React 18 when we do this. I tried doing this update and encountered some warnings about `async` `useEffect` hooks. It's related to this issue because those hooks are where we do data-fetching from APIs. https://github.com/facebook/react/issues/14326

**Comment by dcdenu4** [comment]
This seems done or at least outdated. I didn't want to close because @davemfish you had a comment about upgrading to React 18?
